sites/partners/src/listings/PaperListingForm/sections/ListingIntro.tsx

```diff
@@ -20,25 +20,27 @@ const ListingIntro = (props: ListingIntroProps) => {
   // eslint-disable-next-line @typescript-eslint/unbound-method
   const { register, clearErrors, errors } = formMethods
 
-  const hideSelect = props.jurisdictionOptions.length <= 2
+  const defaultJurisdiction =
```

Comment:
I'm reading this as "If a user has exactly two jurisdictions, then set the default as the second one" but I'm not following why - why is it not "If a user has exactly one jurisdiction, then set the default as that jurisdiction"?

Reply:
jurisdictionOptions is initialized in index.tsx with an empty object first and then the profile's jurisdictions are mapped into the prop. So a length of one would indicate no jurisdictions and a length of two contains the one default jurisdiction.

Reply:
Ahhhh okay I see, that's the empty select option got it. I think it would be cleaner here to not build the select's options inside this component's `jurisdictionOptions` prop, but just to pass in the user's jurisdictions as a prop and have this component decide internally what it wants the select options to be, if that makes sense?

Reply:
Yes, I think so! Just pushed a new commit and believe it reflects what you're saying